docs/config.rst

```diff
@@ -243,6 +243,25 @@ Features
 
    .. versionadded:: 2.0
 
+.. attribute:: LP_ENABLE_CONTAINER
+   :type: bool
+   :value: 0
+
+   Indicate if the shell is running in a container environment (e.g. Docker, 
+   Podman, LXC, Singularity).
+
+   .. note::
+      Containers may inherit some or even no variables from their parent shell,
+      so this may behave inconsisently with different container software.  For 
+      example, Docker doesn't inherit anything unless explicitly told to.  
+      Singularity in many configurations inherits most variables but bash 
```

Comment:
Change "bash" to "shell". Both Bash and Zsh use the same functions, and if we expand to other shells like Ksh, this should be more general.
```suggestion
      Singularity in many configurations inherits most variables but shell
```